src/wasm/wasm-emscripten.cpp

```diff
@@ -229,6 +237,7 @@ ensureFunctionImport(Module* module, Name name, std::string sig) {
 // Here we internalize all such wasm globals and generte code that sets their
 // value based on the result of call `g$foo` and `fp$bar` functions at runtime.
 Function* EmscriptenGlueGenerator::generateAssignGOTEntriesFunction() {
+  BYN_TRACE("generateAssignGOTEntriesFunction\n");
```

Comment:
unrelated to this PR, maybe we could always emit a newline in `BYN_TRACE` so every command doesn't need to?

Reply:
Ah I see there is another comment on just this :)